src/interceptors/XMLHttpRequest/XMLHttpRequestOverride.ts

```diff
@@ -422,13 +422,7 @@ export const createXMLHttpRequestOverride = (
       const headerValue = Object.entries(this.responseHeaders).reduce<
         string | null
       >((_, [headerName, headerValue]) => {
-        // Ignore header name casing while still allowing to set response headers
-        // with an arbitrary casing (no normalization).
```

Comment:
This gives me pause - it looks like it was a conscious decision _not_ to normalize case so I might be missing something here?

Reply:
I think you're correct: header names must be case-insensitive when retrieving a header. We forgot to normalize the input `name` to provision that.